storage/persistent_storage/persistent_storage_local/ways-to-provision-local-storage.adoc

```diff
@@ -0,0 +1,64 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ways-to-provision-local-storage"]
+= Local storage overview
+include::_attributes/common-attributes.adoc[]
+:context: ways-to-provision-local-storage
+
+toc::[]
+
+You can use any of the following solutions to provision local storage:
+
+* HostPath Provisioner (HPP)
+* Local Storage Operator (LSO)
+* {lvms-first}
+
+[WARNING]
+====
+These solutions support provisioning only node-local storage. The workloads are bound to the nodes that provide the storage. If the node becomes unavailable, the workload also becomes unavailable. To maintain workload availability despite node failures, you must ensure storage data replication through active or passive replication mechanisms.
+====
+
+[id="overview-of-hpp-functionality"]
+== Overview of HostPath Provisioner functionality
+
+You can perform the following actions using HostPath Provisioner (HPP):
+
+* Map the host filesystem paths to storage classes for provisioning local storage.
+* Statically create storage classes to configure filesystem paths on a node for storage consumption.
+* Statically provision Persistent Volumes (PVs) based on the storage class.
+* Create workloads and PersistentVolumeClaims (PVCs) while being aware of the underlying storage topology.
+
+[NOTE]
+====
+HPP is available in upstream Kubernetes. However, it is not recommended to use HPP from upstream Kubernetes.
+====
+
+[id="overview-of-lso-functionality"]
+== Overview of Local Storage Operator functionality
+
+You can perform the following actions using Local Storage Operator (LSO):
+
+* Assign the storage devices (disks or partitions) to the storage classes without modifying the device configuration.
+* Statically provision PVs and storage classes by configuring the `LocalVolume` custom resource (CR).
+* Create workloads and PVCs while being aware of the underlying storage topology.
+
+[NOTE]
+====
+LSO is developed and delivered by Red Hat.
+====
+
+[id="overview-of-lvm-storage-functionality"]
+== Overview of LVM Storage functionality
+
+You can perform the following actions using {lvms-first}:
+
+* Configure storage devices (disks or partitions) as lvm2 volume groups and expose the volume groups as storage classes.
+* Create workloads and request storage by using PVCs without considering the node topology.
+
+{lvms} uses the TopoLVM CSI driver to dynamically allocate storage space to the nodes in the topology and provision PVs.
+
+[NOTE]
+====
+{lvms} is developed and maintained by Red Hat. The CSI driver provided with LVMS is the upstream project link:https://github.com/topolvm/topolvm[topolvm].
```

Comment:
Per our [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites) we can't link to external GitHub repositories (I believe for maintainability reasons, like if repo structures change). Up to you how you want to handle this, but maybe the simplest way would be to reference the project by name and hope users would know how to look it up:
```suggestion
{lvms} is developed and maintained by Red Hat. The CSI driver provided with LVMS is the upstream project "topolvm".
```

If there's any non-github doc you can link to for this project, that might be another option.

Reply:
Non-github doc not available for the topolovm project.
Updated as per the suggestion!